stl/inc/cvt/sjis_0208

```diff
@@ -57,7 +57,7 @@ namespace stdext {
                         // convert 2-byte code
                         unsigned char _By2 = static_cast<unsigned char>(*++_Mid1);
 
-                        if ((_By2 < 0x40 || 0x7e < _By2) && (_By2 < 0x80 && 0xfc < _By2)) { // TRANSITION, VSO-609129
+                        if (_By2 < 0x40 || _By2 == 0x7f || _By2 > 0xfc) {

```

Comment:
Thanks for fixing this! (Microsoft-internal VSO-609129 "`<cvt/sjis_0208>` appears to have impossible condition".)

While making this behavioral change as part of a warning cleanup is fine (especially if Clang was warning about a condition that can never be true), it should be specifically called out in the checkin notes.